packages/xstate-react/src/useInterpret.ts

```diff
@@ -93,15 +72,13 @@ export function useInterpret<TMachine extends AnyStateMachine>(
     context,
     guards,
     actions,
+    activities,
```

Comment:
Why do we add this back in?

Reply:
Those were handled before too ([here](https://github.com/statelyai/xstate/pull/2939/files#diff-3efb60b28006844ac5e7d12c3877125205eef496823208e94488eadb16792942L103)) - but the type-level plumbing has changed slightly here and we no longer need to access this property separately.